site/vi/README.md

```diff
@@ -3,10 +3,17 @@
 ## Bản dịch Cộng đồng
 
 Cộng đồng TensorFlow tại Việt Nam đã và đang dịch những tài liệu này từ nguyên bản tiếng Anh.
-Những bản dịch này được hoàn thiện dựa trên sự nỗ lực đóng góp từ cộng đồng lập trình viên sử dụng TensorFlow, và có thể không đảm bảo được tính cập nhật của bản dịch đối với [Tài liệu chính thức bằng tiếng Anh](https://www.tensorflow.org/?hl=en) này. Nếu bạn có bất kỳ đề xuất nào nhằm cải thiện bản dịch này, vui lòng tạo Pull request đến kho chứa trên GitHub của [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n). Để đăng ký dịch hoặc cải thiện nội dung bản dịch, các bạn hãy liên hệ và đặt vấn đề tại 
-[docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
-
-**Lưu ý:** Chỉ tập trung mọi cơ hội có thể dịch thuật vào nội dung [TensorFlow 2](https://www.tensorflow.org) này trong đường dẫn [site/en](https://github.com/tensorflow/docs/tree/master/site/en). Tài liệu cộng đồng TF 1.x trong [site/en/r1](https://github.com/tensorflow/docs/tree/master/site/en/r1) sẽ không còn được cập nhật khi chúng tôi chuẩn bị ra mắt phiên bản 2.0. Xem thêm thông báo [tại đây](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+Những bản dịch này được hoàn thiện dựa trên sự nỗ lực đóng góp từ cộng đồng lập trình viên sử dụng TensorFlow, 
```

Comment:
If changing the disclaimer text, we will need to update it in the notebooks: https://github.com/tensorflow/docs-l10n/tree/master/site/vi/tutorials

Reply:
Sure, I will update this information.

Reply:
@lamberta I would like to ask that this disclaimer text always appears in all documents translated by the Vietnamese community, right?